Jenkinsfile

```diff
@@ -17,12 +17,14 @@ pipeline {
     disableResume()
     durabilityHint('PERFORMANCE_OPTIMIZED')
     disableConcurrentBuilds()
+//    checkoutToSubdirectory "${env.BASE_DIR}"
   }
   triggers {
     issueCommentTrigger('(?i).*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tests(?:\\W+please)?.*')
   }
   parameters {
     booleanParam(name: 'runAllStages', defaultValue: false, description: 'Allow to run all stages.')
+    booleanParam(name: 'windowsTest', defaultValue: false, description: 'Allow to run all stages.')
```

Comment:
Description looks like it needs updated.

Reply:
```suggestion
    booleanParam(name: 'windowsTest', defaultValue: false, description: 'Allow Windows stages.')
```